Geometry/CMSCommonData/python/cmsExtendedGeometry2017XML_cfi.py

```diff
@@ -234,8 +234,8 @@
         'Geometry/MuonCommonData/data/rpcf/2015/v1/rpcf.xml',
         'Geometry/MuonCommonData/data/csc/2015/v1/csc.xml',
         'Geometry/MuonCommonData/data/mfshield/2017/v1/mfshield.xml',
-        'Geometry/MuonCommonData/data/PhaseII/gemf.xml',
-        'Geometry/MuonCommonData/data/PhaseII/gem11.xml',
+        'Geometry/MuonCommonData/data/gemf/TDR_BaseLine/gemf.xml',
+        'Geometry/MuonCommonData/data/gem11/Run2/gem11.xml',
```

Comment:
@bsunanda - Note, that from now on modifications to gemf.xml or gem11.xml will be rejected. Please, move them to a versioned structure to allow further modifications. For example: gem11/2017/v1/gem11.xml